programs/perpetuals/tests/instructions/mod.rs

```diff
@@ -0,0 +1,21 @@
+pub mod test_add_custody;
```

Comment:
is it possible to get rid of mod.rs files and use a newer convention? similar to the main program files structure?